src/components/Search.vue

```diff
@@ -1,14 +1,12 @@
 <template lang="pug">
-v-layout.searchbar(row, justify-center, align-center)
+v-layout.girder-searchbar(row, align-center)
   v-icon.mdi-24px.mx-2(color="white") {{ $vuetify.icons.search }}
-  v-text-field.searchtext.mx-4(
-      light, solo, hide-details, clearable,
-      v-model="searchtext")
-  v-menu.searchbar(
-      offset-y, max-width,
-      :activator="'.searchtext'",
+  v-menu.searchbar.grow(
```

Comment:
Please add `content-class="girder-searchbar"` to this v-menu and remove `.searchbar`.  The searchbar style has no effect.

Reply:
There were three .searchbar from the beginning, I later renamed the top level one. I am thinking to remove the .searchbar off the two v-menu, will that functionally work?